tests/InputFiles/GeneralizedHarmonic/KerrSchild.yaml

```diff
@@ -0,0 +1,50 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Executable: EvolveGeneralizedHarmonic
+# Check: execute
+# Timeout: 4
```

Comment:
Reducing the number of grid points doesn't significantly change the run time, nor does reducing the final time. The only thing I found that reduces the run time at all is going from 1 refinement to 0 refinements, but i) that's not great, because then there are not interior boundaries in the test, and ii) it still only reduces the run time by about 1/3, whereas to get from 4s to 2s I would need a reduction by a factor of 1/2. So I think for now we should leave the timeout at 4.

Reply:
Do you know, what makes this test so slow?

Reply:
What I've seen is that with debugging symbols enabled (i.e. in `Debug` mode), the run time of this test increases by a factor of 4+. My best guess is that loading these symbols initially during execution takes most of the extra time, especially due to the large number of compute tags involved.